__tests__/gas-refund-program/__snapshots__/safety-module-stakes-tracker.ts.snap

```diff
@@ -150,3 +150,61 @@ exports[`SafetyModuleStakesTracker snashot test for backward compat Init state:
   \\"bptPoolTotalSupply\\": \\"486095993118680093200539\\"
 }"
 `;
+
+exports[`SafetyModuleStakesTracker snashot test for backward compat Init state: SafetyModuleStakesTracker::initState at block 14567001 1`] = `
+"{
+  \\"stkPSPBptUsersBalances\\": {
+    \\"0x0ddc793680ff4f5793849c8c6992be1695cbe72a\\": \\"128179093238755885821351\\",
```

Comment:
old test [used to set startBlock](https://github.com/paraswap/paraswap-volume-tracker/pull/60/files#diff-121ca613c66071276e0562ecb19e43d3286ce351bdbf2d8fe7c10b2eb3e9bc18L18) to startBlock+1 hence diff stakes. I think this isn't relevant